opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/SpanShim.java

```diff
@@ -183,7 +183,28 @@ public void finish(long finishMicros) {
     span.end(finishMicros, TimeUnit.MICROSECONDS);
   }
 
-  static String getEventNameFromFields(Map<String, ?> fields) {
+  private void logInternal(long timestampMicroseconds, Map<String, ?> fields) {
+    String name = getEventNameFromFields(fields);
+    boolean isExceptionEvent = false;
+    Throwable throwable = null;
+    if (name.equals(ERROR)) {
+      throwable = findThrowable(fields);
+    } else if (name.equals(EXCEPTION)) {
```

Comment:
I can't find this name `exception` in the spec

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/compatibility/opentracing.md#log

As far as I can tell, we only have "error event", and that error event may or may not have a throwable.

So

```java
boolean isError;
boolean recordedException;
if (getEventNameForFields == "error") {
  isError = true;
  if (throwable != null) {
    recordException()
    recordedException = true;
  }

...

// Filling attributes
if (recordedException) {
  if (key.equals(ERROR_OBJECT)) {
    continue;
  }
  // Don't see anything in spec to skip ERROR_KIND, etc for recordedException case like current code does, should we make clearer in the spec if we think it should be skipped?
}
if (isError) {
  if (key.equals(ERROR_KIND)) {
    key = EXCEPTION_TYPE;
  }
  ...
}
```

Reply:
Next paragraph is confusing, but it says that name being set to `exception`:
 
> Else, a call to AddEvent is performed with name being set to exception, along the specified key/value pair set as additional event attributes, including mapping of the following key/value pairs:

Reply:
Yeah that's the name of the OpenTelemetry event, but is still in the paragraph for `If pair set contains a event=error entry`

Reply:
updated